### ISSUE TITLE
1.0.9 release plans?

### ISSUE THREAD

**Issue by craigcitro** [keras-team/keras-preprocessing#162]
tl;dr: Is there a plan for a 1.0.9 release?

We automatically include keras-preprocessing in Google Colab, where we had a user run into https://github.com/googlecolab/colabtools/issues/408 -- this looks to already be fixed at HEAD, and I'm going to bump us back to 1.0.6 for now, but I wanted to see if there was a plan for 1.0.9 (since I suspect other users will also be broken).

- [x] Check that you are up-to-date with the master branch of keras-preprocessing. You can update with:
`pip install git+git://github.com/keras-team/keras-preprocessing.git --upgrade --no-deps`

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Repro link: https://colab.research.google.com/drive/1XIBXpR--3PPNPypBDfhim3qwZla7Zkwp

**Comment by Dref360** [comment]
Kinda weird that this was not caught by our tests. 

I'll make a release asap.

**Comment by Dref360** [comment]
DOne
